devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy

```diff
@@ -92,10 +88,24 @@ def setLevel(value) {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig() + zigbee.levelConfig() + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    return zigbee.readAttribute(0x0006, 0x0000) +
+           zigbee.readAttribute(0x0008, 0x0000) +
+           zigbee.readAttribute(0x0702, 0x0400) +
+           zigbee.readAttribute(0x0B04, 0x050B) +
+           zigbee.configureReporting(0x0006, 0x0000, 0x10, 0, 600, null) +
+           zigbee.configureReporting(0x0008, 0x0000, 0x20, 1, 3600, 0x01) +
+           zigbee.configureReporting(0x0702, 0x0400, 0x2A, 1, 600, 0x05) +
+           zigbee.configureReporting(0x0B04, 0x050B, 0x29, 1, 600, 0x0005)
```

Comment:
1. The `*Refresh` commands are not documented so we were going to change the code to only use documented calls.
2. I haven't seen that problem. I can try this code again but I'm pretty sure it works fine.